src/lamp_py/runtime_utils/env_validation.py

```diff
@@ -55,7 +58,9 @@ def validate_environment(
                 # do not log private variables
                 if key in private_variables:
                     value = "**********"
-                process_logger.add_metadata(**{key: value})
+                metadata[key] = value
+
+    process_logger.add_metadata(**metadata)
```

Comment:
adding the metadata for each key was spamming the logs, so just do it once with all the metadata.